src/interactions/interaction.ts

```diff
@@ -24,16 +25,18 @@ export class Interaction {
    * @returns {Interaction} The calling Interaction.
    */
   public attachTo(component: Component) {
-    if (this._componentAttachedTo) {
-      this.detachFrom(this._componentAttachedTo);
-    }
-
+    this._disconnect();
     this._componentAttachedTo = component;
-    component.onAnchor(this._anchorCallback);
-
+    this._connect();
     return this;
   }
 
+  private _connect() {
```

Comment:
Unsure if I like the name.  My first response is ok `interaction._connect()` will connect to...?

That said, this is a private function and it alludes to what it's doing sorta somewhat ok.  Not a huge deal.